app/routes/__index/dns-records/$dnsRecordId.tsx

```diff
@@ -42,7 +42,8 @@ export const action = async ({ request }: ActionArgs) => {
       return isNameValid(fqdn, user.username);
     },
     {
-      message: 'Record name is invalid',
+      message:
+        'Record name is invalid or inappropriate (only alphanumerical character, -, and _ allowed)',
```

Comment:
Is it possible to add on the message the specific value that is considered inappropriate?

Reply:
Currently it uses backend function that returns `boolean`. To show specific error message, it takes much bigger scope. I will write another issue to follow up if that is acceptable.

Reply:
It might be better to be vague.

Reply:
I was thinking some users try to use special characters, but they don't know why it's invalid. So I tried to at least give what's allowed. Should it be just `Record name is invalid or inappropriate`?

Reply:
We can provide the information on the tooltip no?